client/src/components/Minimap/Minimap.scss

```diff
@@ -1,24 +1,21 @@
-// If unset, default to the "mobile" height of the slim header.
-$minimap-top-offset: var(--offset-top, calc(theme('spacing.slim-header') * 2));
```

Comment:
There is really no reason for the variable to be unset, so I’ve removed that default for simplicity’s sake.